pkg/apis/monitoring/v1/types.go

```diff
@@ -426,6 +432,24 @@ type PrometheusRuleExcludeConfig struct {
 	RuleName string `json:"ruleName"`
 }
 
+// PrometheusObjectReference prometheus object reference
+type PrometheusObjectReference struct {
+	// API version of the referent.
+	APIVersion string `json:"apiVersion,omitempty"`
+	// Kind of the referent.
+	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
+	Kind string `json:"kind,omitempty"`
```

Comment:
We could have some validation on the value (e.g. `PodMonitor`, `ServiceMonitor` or `PrometheusRule`)

Reply:
Done